diff --git a/dist/Decorator.jsx b/dist/Decorator.jsx
index 9fa73ed9010697067054f742f46574167696c05f..7ce8cbdcf594c5fa939b7fd379393908f0b4fe7b 100644
--- a/dist/Decorator.jsx
+++ b/dist/Decorator.jsx
@@ -1,7 +1,9 @@
 import React from 'react';
 import { getDecoratorRsc } from './fetch-decorator-next';
 export async function Decorator({ children, decoratorProps }) {
+    console.time("RSC: Decorator")
     const { Styles, Header, Footer, Scripts } = await getDecoratorRsc(decoratorProps);
+    console.timeEnd("RSC: Decorator")
     return (<>
             <head>
                 <Styles />
